### ISSUE TITLE
Additional Freezer Position Form Request

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#1729]
Submitting separate issue per Dusty's request in #1586 

Reviving because this is becoming a roadblock to a grant-funded project.

I'd like to step back and keep this very simple for the moment. Let's start with the freezer positions as seen [here](http://arctos-test.tacc.utexas.edu/containerPositions.cfm?container_id=18085477).

This form does not reflect the actual positions of racks in our freezers. It may do so for other collections since no one else is complaining (@KyndallH ?), but we would like a 33 position freezer form that reflects the actual positions in our freezers. Which look like this:
![image](https://user-images.githubusercontent.com/5725767/46684565-6bea4d00-cbb0-11e8-898e-387cf3474c7b.png)
So, the form should look like this:
![image](https://user-images.githubusercontent.com/5725767/46684626-a05e0900-cbb0-11e8-807c-c376fb5865bb.png)

Matching the "map" of positions in Arctos to the physical positions in the freezer makes it easier for people (students in particular) to find a rack.

We would like this resolved by October 25, 2018

**Comment by dustymc** [comment]
Thanks.

You are asking to change this:

```

UAM@ARCTOS> select count(*) from container p, container f where p.parent_container_id=f.container_id and p.container_type='position' and f.container_type='freezer';

  COUNT(*)
----------
       968
```

for these containers:

```
UAM@ARCTOS> select f.INSTITUTION_ACRONYM, count(*) from container p, container f where p.parent_container_id=f.container_id and p.container_type='position' and f.container_type='freezer' group by f.INSTITUTION_ACRONYM;

INSTITUTION_ACRONYM					       COUNT(*)
------------------------------------------------------------ ----------
UAM								    671
MSB								    297

2 rows selected.
```

I'm pretty sure UAM is using the horizontal orientation, but if they're not I can change it.

If they don't want to not-quite-randomly shuffle their freezer locations, we can figure something out.

**Comment by Jegelewicz** [comment]
Got it. We can discuss along with freezer rack issue when we meet up.

**Comment by dustymc** [comment]
UAM isn't using, just change it

**Comment by dustymc** [comment]
blocked by https://github.com/ArctosDB/arctos/issues/1743 (this may not be necessary)

**Comment by dustymc** [comment]
We're proceeding with  #1743 so this is irrelevant.
